openadapt/ml/2_processed.jsonl

```diff
@@ -0,0 +1,356 @@
+{"prompt": "({'name': 'move', 'mouse_x': 949.84765625, 'mouse_y': 629.609375, 'element_state': {}}, {'title': 'Terminal ml \u2014 poetry shell \u25b8 Python \u2014 204\u00d742', 'left': 2, 'top': 86, 'width': 1438, 'height': 623, 'window_id': 169139})", "completion": " ({'name': 'move', 'mouse_x': 949.23046875, 'mouse_y': 628.68359375, 'element_state': {}}, {'title': 'Terminal ml \u2014 poetry shell \u25b8 Python \u2014 204\u00d742', 'left': 2, 'top': 86, 'width': 1438, 'height': 623, 'window_id': 169139})"}
```

Comment:
@FFFiend can you please clarify how this file was generated? Can we generate it programmatically?

Reply:
Not necessary, I just included it since this was the first draft and I wanted everyone to see what the file looked like. Will delete 👍